### PR TITLE
Add links to all tools in main README

### PR DESCRIPTION
**Pull request by pudelkoM**
This PR adds some missing links to tools we provide to increase their visibility. Not all of them have READMEs, sadly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,13 @@ If you're completly new to the world of P4 and are looking for step-by-step inst
 ### Tools
 
 - [gNMI CLI Tool](/stratum/tools/gnmi/README.md)
+- [BF Pipeline Builder](/stratum/hal/bin/barefoot/README.pipeline.md#)
```
Comment (pudelkoM) [inline README.md:52]: fix link to section

File: README.md [hunk 2]
```diff
@@ -49,8 +49,13 @@ If you're completly new to the world of P4 and are looking for step-by-step inst
...
 - [Stratum-Enabled Mininet](/tools/mininet/README.md)
 - [P4Runtime write request replay tool](/stratum/tools/stratum-replay/README.md)
+- [Bf ChassisConfig Migrator](/stratum/hal/config/chassis_config_migrator.cc)
```
Comment (pudelkoM) [inline README.md:55]:
```suggestion
- [ChassisConfig Migrator](/stratum/hal/config/chassis_config_migrator.cc)
```

File: README.md [hunk 3]
```diff
@@ -49,8 +49,13 @@ If you're completly new to the world of P4 and are looking for step-by-step inst
 ### Tools
 
 - [gNMI CLI Tool](/stratum/tools/gnmi/README.md)
+- [Bf Pipeline Builder](/stratum/hal/bin/barefoot/README.pipeline.md#stratum-bfpipelineconfig-format-and-the-bfpipelinebuilder)
```
Comment (Yi-Tseng) [inline README.md:52]: Maybe use `BF` or `Barefoot` ?

Comment (pudelkoM) [inline README.md:52]:
```suggestion
- [Tofino Pipeline Builder](/stratum/hal/bin/barefoot/README.pipeline.md#stratum-bfpipelineconfig-format-and-the-bfpipelinebuilder)
```

Comment (pudelkoM) [reply README.md:52]: Good point, I'll go with Tofino since Barefoot is Intel now/soon.

**Review by Yi-Tseng — APPROVED** [review]

**Pull request merged**